Bio/SeqAnalysis/SeqDatabase.py

```diff
@@ -0,0 +1,277 @@
+
+        total_number = len(only_files)
+        print("Download of " + str(total_number) + f" took {timer() - start} seconds")
+
+        print("-------------------------------------Saving outputs------------------------------------------")
+        print('Possibly obsolete entries were saved in Failed/{0}/Obsolete.txt'.format(analysis_dir))
+        print('Number of obsolete entries: {0}'.format(str(len(obsolete))))
+
+        with open('./Failed/{0}/Obsolete.txt'.format(analysis_dir), 'w+') as obs:
+            for item in obsolete:
+                obs.write(item + '\n')
+
+        print('Entries with possible download error were saved in Failed/{0}/Error.txt'.format(analysis_dir))
+        print('Number of download error entries: {0}'.format(str(len(http_error))))
+
+        with open('./Failed/{0}/Error.txt'.format(analysis_dir), 'w+') as fail:
+            for item in http_error:
+                fail.write(item + '\n')
+
+        print("Creating database. Please wait...")
+        for file in only_files:
+            with open('./Downloads/{0}/{1}'.format(analysis_dir, file)) as big_fasta:
+                one_fasta = big_fasta.read()
+                with open('./Database/{0}.fasta'.format(analysis_dir), 'a+') as data:
+                    data.write(one_fasta + '\n')
+
+        print("Your sequences were saved in Database/{0}.fasta".format(analysis_dir))
+        print("Done.")
+        self.downloaded_files = files_no_ext
+
+    @staticmethod
+    def read_list_of_ids(file: str) -> List:
+        """
+        Reads given file and creates List with unique IDs.
+
+        :param file: str, name of file.
+        :raise: Exception when can't find a file.
+        :return: List with IDs.
+        """
+        try:
+            data = ''
+            with open(file, 'r') as f:
+                for line in f.readlines():
+                    line = re.sub('[!@/#$|:]', ' ', line).strip()
+                    # Substitutes above signs with space ensuring your
+                    # file with IDs will be always read correctly.
+                    data = data + ' ' + line
+                data = data.split()
+                data = list(dict.fromkeys(data))
+            return data
+        except FileNotFoundError:
+            raise Exception("!!!ERROR!!! \n"
+                            "Please enter right name of file (with .txt).\n"
+                            "Remember that your ID file should be in the same directory as this script.\n"
+                            "If not please specify path to the ID file.\n"
+                            "E.g. dir1/subdir1/list.txt")
+
+    @staticmethod
+    def mk_dirs(dir_name: str):
+        if not os.path.exists(dir_name):
+            os.mkdir(dir_name)
+            print("Directory {0} status: created".format(dir_name))
+        else:
+            print("Directory {0} status: already exists".format(dir_name))
```

Comment:
For working with files great library is (https://docs.python.org/3/library/pathlib.html)[pathlib], but of course, using `os` is not an issue